content/id/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,374 @@
+yang menentukan kelas isolasi yang tersedia, bagian dari konkurensi anggaran yang tersedia 
+yang masing-masing dapat menangani bagian tersebut, dan memperbolehkan untuk melakukan _fine-tuning_ terhadap perilaku antrian.
+[FlowSchema](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#flowschema-v1alpha1-flowcontrol-apiserver-k8s-io)
+yang digunakan untuk mengklasifikasikan permintaan individu yang masuk, mencocokkan masing-masing dengan setiap 
+PriorityLevelConfiguration.
+
+### PriorityLevelConfiguration
+Sebuah PriorityLevelConfiguration merepresentasikan sebuah kelas isolasi tunggal. Setiap
+PriorityLevelConfiguration memiliki batas independensi dalam hal jumlah
+permintaan yang belum diselesaikan, dan batasan dalam hal jumlah permintaan yang mengantri.
+
+Batas konkurensi untuk PriorityLevelConfiguration tidak disebutkan dalam
+jumlah permintaan secara mutlak, melainkan dalam "concurrency shares." Total batas konkurensi
+untuk Server API didistribusikan di antara PriorityLevelConfiguration yang ada 
+secara proporsional dengan "concurrency shares" tersebut. Ini mengijinkan seorang
+administrator klaster untuk meningkatkan atau menurunkan jumlah total lalu lintas ke sebuah
+server dengan memulai kembali `kube-apiserver` dengan nilai opsi
+`--max-request-inflight` (atau `--max-mutating-request-inflight`) yang berbeda, dan semua
+PriorityLevelConfiguration akan melihat konkurensi maksimum yang diizinkan kepadanya untuk menaikkan (atau
+menurunkan) dalam fraksi yang sama.
+
+{{< caution >}}
+Dengan fitur Prioritas dan Kesetaraan yang diaktifkan, batas total konkurensi untuk
+server diatur pada nilai penjumlahan dari `--max-request-inflight` dan
+`--max-mutating-request-inflight`. Tidak akan ada lagi perbedaan
+antara permintaan yang bermutasi dan permintaan yang tidak bermutasi; jika kamu ingin melayaninya
+secara terpisah untuk suatu sumber daya yang ada, maka perlu membuat FlowSchema terpisah yang sesuai dengan
+masing-masing kata kerja dari permintaan yang bermutasi dan yang tidak bermutasi tersebut.
+{{< /caution >}}
+
+Ketika jumlah permintaan masuk yang diserahkan kepada sebuah
+PriorityLevelConfiguration melebihi dari tingkat konkurensi yang diizinkan, 
+bagian `type` dari spesifikasinya menentukan apa yang akan terjadi pada permintaan selanjutnya.
+Tipe `Reject` berarti bahwa kelebihan lalu lintas akan segera ditolak
+dengan kode kesalahan HTTP 429 (yang artinya terlalu banyak permintaan). Tipe `Queue` berarti permintaan
+di atas batas tersebut akan mengantri, dengan teknik _sharding shuffle_ dan _fair queuing_ yang digunakan
+untuk menyelaraskan kemajuan antara _flow_ permintaan.
+
+Konfigurasi antrian memungkinkan mengatur algoritma _fair queuing_ untuk sebuah
+tingkat prioritas. Detail algoritma dapat dibaca di [proposal pembaharuan](#what-s-next), namun secara singkat:
```

Comment:
`#what-s-next` is a broken anchor link, fix it with the translated one

Reply:
I don't know to link to this one `{{% capture whatsnext %}}`

Reply:
this translates as "selanjutnya", the link works: https://deploy-preview-20588--kubernetes-io-master-staging.netlify.app/id/docs/concepts/cluster-administration/flow-control/#selanjutnya

Reply:
Ok! Fixed.